.github/workflows/tests.yml

```diff
@@ -1,6 +1,14 @@
-name: CI
+name: Tests
 
-on: [ push, pull_request ]
+on:
+  push:
+    branches:
+      - '**'
+      - '!TYPO3V8'
```

Comment:
Why are you ignoring TYPO3V8 branch? If we want a release for the TYPO3v8 version for some reason it would be nice with some tests. Or do you still have the old tests.yaml (ci.yaml) in the TYPO3V8 branch? If so ignore my comment.

Reply:
I had to create an own TYPO3V8 branch instead of non compatibility changes in SchedulerModuleController. TYPO3V8 has its own tests.yml.